src/app/containers/RadioPageBlocks/Blocks/OnDemandHeading/__snapshots__/index.test.jsx.snap

```diff
@@ -89,8 +89,6 @@ exports[`AudioPlayer blocks OnDemandHeading should render correctly 1`] = `
 
 <h1
   class="c0"
-  id="content"
```

Comment:
This should be added back with tabindex too @ryanmccombe ?

Reply:
This component no longer applies id=content and tab-index=-1 by default

It only does this is the page passes in "content" as the id attribute

The radio page is doing that, so will continue to work as expected - the radio snapshots are here: https://github.com/bbc/simorgh/blob/latest/src/app/pages/OnDemandRadioPage/__snapshots__/index.test.jsx.snap

They're not updated in this PR, because this PR doesn't change the behavior of the radio page